elements/pf-popover/pf-popover.ts

```diff
@@ -54,10 +54,11 @@ export class PopoverShownEvent extends ComposedEvent {
 }
 
 /**
- * Patternfly popover
- *
  * A Popover displays content in a non-modal dialog and adds contextual information or provides resources via text and links.
  *
+ * Unlike the Patternfly React pattern, this component does not trap focus in the popover dialog.
```

Comment:
I think we should move this to the docs site contents, as the content of `renderOverview`

Reply:
Gotcha. On it!